Resources/doc/reference/security.rst

```diff
@@ -112,7 +112,7 @@ Let's create the following strategy : a media can be downloaded only once per se
                 return false;
             }
 
-            $this->getSession()->set($this->sessionKey, $times++);
+            $this->getSession()->set($this->sessionKey, ++$times);
```

Comment:
There is no need to increment `$times`, is there? So maybr write `$times + 1`. Good catch anyway!